assets/src/common/helpers/test/get-plugin-slug-from-path.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Internal dependencies
+ */
+import { getPluginSlugFromPath } from '../get-plugin-slug-from-path';
+
+describe( 'getPluginSlugFromPath', () => {
+	it( 'should return an empty string if no path is provided', () => {
+		expect( getPluginSlugFromPath() ).toBe( '' );
+	} );
+
+	it( 'should return correct plugin slug', () => {
+		expect( getPluginSlugFromPath( 'foo' ) ).toBe( 'foo' );
+		expect( getPluginSlugFromPath( 'foo.php' ) ).toBe( 'foo' );
+		expect( getPluginSlugFromPath( 'foo/bar' ) ).toBe( 'bar' );
+		expect( getPluginSlugFromPath( 'foo/baz.php' ) ).toBe( 'baz' );
```

Comment:
These don't seem quite right. See corresponding PHP test:

https://github.com/ampproject/amp-wp/blob/041cc29a4c1baa834cb6bab62837e839f1f04858/tests/php/src/PluginRegistryTest.php#L38-L45

```suggestion
		expect( getPluginSlugFromPath( 'foo/foo.php' ) ).toBe( 'foo' );
		expect( getPluginSlugFromPath( 'foo/extra.php' ) ).toBe( 'foo' );
		expect( getPluginSlugFromPath( 'foo.php' ) ).toBe( 'foo.php' );
```